components/pango_display/src/view.cpp

```diff
@@ -312,7 +312,8 @@ GLfloat View::GetClosestDepth([[maybe_unused]] int x, [[maybe_unused]] int y, in
 #endif
 
     const int zsize = zl*zl;
-    GLfloat zs[zsize];
+    std::vector<GLfloat> zsVec(zsize);
+    GLfloat* zs = zsVec.data();
```

Comment:
Why do you allocate the memory through a `std::vector` and not `malloc`/`free`? Also, when you turn this into a dynamic array, you can remove the `MSVC` workaround above.